api/Window.json

```diff
@@ -866,10 +866,16 @@
               "version_added": true
             },
             "firefox": {
-              "version_added": true
+              "version_added": true,
+              "notes": [
+                "Before version 62 this function returns <code>null</code> when called on a <code>Window</code> which has no presentation, such as an <code>iframe</code> with <code>display: none;</code>. From version 62 onwards it returns an empty string in this situation."
```

Comment:
`getComputedStyle` doesn't return an empty string, it just doesn't return null. `getPropertyValue` would return an empty string. I'd drop everything after `From version [..]`